studio/components/layouts/ProjectLayout/ProjectPausedState.tsx

```diff
@@ -128,11 +136,21 @@ const ProjectPausedState = ({ product }: ProjectPausedStateProps) => {
                     </Tooltip.Portal>
                   )}
                 </Tooltip.Root>
-                <Link href={`/project/${ref}/settings/general`}>
-                  <a>
-                    <Button type="default">View project settings</Button>
-                  </a>
-                </Link>
+                {isFreePlan ? (
+                  <Link
+                    href={`/project/${ref}/settings/billing/subscription?panel=subscriptionPlan`}
+                  >
+                    <a>
+                      <Button type="default">Change Plan</Button>
```

Comment:
No hard feelings, just food for thought to think about different CTAs for this one

"Upgrade Plan"
"Upgrade to Pro"

Can keep it as it is though

Reply:
Good idea will change